src/server/typingsInstaller/typingsInstaller.ts

```diff
@@ -332,6 +338,9 @@ namespace ts.server.typingsInstaller {
                     if (this.log.isEnabled()) {
                         this.log.writeLine(`Installed typings ${JSON.stringify(scopedTypings)}`);
                     }
+
+                    this.writePackageVersion(cachePath);

```

Comment:
that said, this is a good change by itself. my concern is this setting the version in the install action is not safe per se; you could have other packages already installed that do not match the version, so just setting the version tricks you on next load to think they are all updated where that is not the case. removing all of them is guaranteed to be correct all the time.

what is the scenario you are trying to enable?